packages/api-specification/api/windows.ts

```diff
@@ -0,0 +1,313 @@
+import {Bounds, UnsubscribeFunction} from "./common";
+import {Base64ImageData, CaptureOptions} from "./system";
+import EventEmitter = NodeJS.EventEmitter;
+
+export interface WindowsAPI {
+
+    /** Returns the current window */
+    current: Window;
+
+    /** Returns all windows in the application */
+    all: Window[];
+
+    /**
+     * The window that is focused in this application, otherwise returns null
+     */
+    focusedWindow: Window;
+
+    /**
+     getById(windowId: string): Window;
+
+     /** Opens a new window */
+    open(options: WindowOptions): Promise<Window>;

```

Comment:
I prefer using the constructor pattern that is used in some of the newer browser APIs (like notifications). Might be better to keep it like this though if `window.open` is used in the symphony application.